DatadogTrace/Sources/OpenTelemetry/OTelSpan.swift

```diff
@@ -86,23 +86,54 @@ internal class OTelSpan: OpenTelemetryApi.Span {
         )
     }
 
-    // swiftlint:disable unavailable_function
+    /// Sends a span event which is akin to a log in Datadog
+    /// - Parameter name: name of the event
     func addEvent(name: String) {
-        fatalError("Not implemented yet")
+        addEvent(name: name, timestamp: .init())
     }
 
+    /// Sends a span event which is akin to a log in Datadog
+    /// - Parameters:
+    ///  - name: name of the event
+    /// - timestamp: timestamp of the event
     func addEvent(name: String, timestamp: Date) {
-        fatalError("Not implemented yet")
+        addEvent(name: name, attributes: .init(), timestamp: .init())
```

Comment:
bug?/ I believe we meant to pass the `timestamp` downstream:
```suggestion
        addEvent(name: name, attributes: .init(), timestamp: timestamp)
```